src/OAuth2/LaminasHttpPhpEnvironmentBridge/Response.php

```diff
@@ -45,8 +45,12 @@ public function setError($statusCode, $name, $description = null, $uri = null)
         );
     }
 
-    public function setRedirect($statusCode = 302, $url, $state = null, $error = null, $errorDescription = null, $errorUri = null)
```

Comment:
This pattern of non-required parameters before required ones is deprecated in PHP 8 (phpstan catches these if the tests don't hit the code path).

I added an if condition below to set the statusCode to 302 if it's falsey.